README.md

```diff
@@ -87,6 +88,34 @@ For further information, please visit the [Airflow Wiki](https://cwiki.apache.or
 
 Official container (Docker) images for Apache Airflow are described in [IMAGES.rst](IMAGES.rst).
 
+## Installing from PyPI
+
+Airflow is published as `apache-airflow` package in PyPi. Installing it however might be sometimes tricky
+because Airflow is a bit of both a library and application. Libraries usually keep their dependencies open and
+Applications usually pin them but we should do neither and both at the same time. We decided to keep
+our dependencies as open as possible (in setup.py) so users are not limited to certain versions of
+dependencies. This means that from time to time `pip install apache-airflow` will not work or will
+produce unusable Airflow installation.
+
+In order to have repeatable installation, however starting from Airlfow 1.10.10 we also keep a set of
+"known-to-be-working" requirements.txt in the requirements folder. Those "known-to-be-working"
+requirements are per Major/Minor python version and you can use them as constraint files when installing
+Airflow from PyPi as follows (note that you have to specify correct Airflow version and python versions
+in the URL:
+
+```bash
+pip install apache-airflow[..EXTRAS HERE..]==1.10.10 \
+ --constraint https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.7.txt
+```
```

Comment:
Can we separate it into two just to avoid confusion among our new users:

1) Just installing Airflow:

```bash
pip install apache-airflow==1.10.10 \
 --constraint https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.7.txt
```

2) Installing with EXTRAS:

```bash
pip install apache-airflow[..EXTRAS HERE..]==1.10.10 \
 --constraint https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.7.txt
```

Example:
```bash
pip install apache-airflow[postgres]==1.10.10 \
 --constraint https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.7.txt
```